tests/Mailer/Ownership/RecipientStrategy/SendToOwnersOfOrganisationTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Mailer\Ownership\RecipientStrategy;
+
+use CultuurNet\UDB3\Model\ValueObject\Identity\Uuid;
+use CultuurNet\UDB3\Ownership\Repositories\OwnershipItem;
+use CultuurNet\UDB3\ReadModel\DocumentRepository;
+use CultuurNet\UDB3\ReadModel\JsonDocument;
+use CultuurNet\UDB3\User\UserIdentityDetails;
+use CultuurNet\UDB3\User\UserIdentityResolver;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+
+class SendToOwnersOfOrganisationTest extends TestCase
+{
+    private MockObject $identityResolver;
```

Comment:
```suggestion
    /**
     * @var UserIdentityResolver&MockObject
     */
    private $identityResolver;
```
Always handy to have better typehinting, same for the other `MockObjects`